mono/mini/exceptions-amd64.c

```diff
@@ -1167,6 +1181,14 @@ terminate_table_no_lock (void)
 			g_ntdll = NULL;
 		}
 
+		g_rtl_delete_function_table = NULL;
+		g_rtl_install_function_table_callback = NULL;
+
+		if (g_kernel32dll != NULL) {
+			FreeLibrary (g_kernel32dll);
```

Comment:
Never FreeLibrary kernel32.dll. It doesn't do anything but waste code and cycles.

Reply:
Also remove the FreeLibrary of ntdll.dll.
Imagine if unloading these .dlls really did anything.
ntdll.dll is loaded specially by the kernel.
ntdll.dll, the .exe, and direct and indirect dependencies of the .exe are special and not unloadable. They do not actually maintain a reference count.

Reply:
Just following the documentation of the GetModuleHandleEx API:

From MSDN:

"dwFlags [in]

This parameter can be zero or one or more of the following values. If the module's reference count is incremented, the caller must use the FreeLibrary function to decrement the reference count when the module handle is no longer needed."

So not doing this for kernel32.dll is based on internal information not described in the above documentation. I also know that this is really not needed in practice, due to internal implementation details, just following the public API documentation of the API.

No big deal, keeping it won't harm, I'm fine to remove it as well.